portal-api/api-tests/v1/gef/eligibility-criteria.api-test.js

```diff
@@ -105,6 +106,7 @@ describe(baseUrl, () => {
         isInDraft: expect.any(Boolean),
         createdAt: expect.any(Number),
         criteria: expect.any(Array),
+        auditRecord: generateParsedMockPortalUserAuditDatabaseRecord(anAdmin._id),
```

Comment:
This test is currently failing. It looks like this submission adds a second `version 2.1` EC to the database( Mock data loader has already added this). When the GET request is made, it returns the first one (i.e. that made my MDL) so the test was passing previously, but not anymore